app/javascript/driving/actions/actionCreators.js

```diff
@@ -1,4 +1,5 @@
-import fetch from 'isomorphic-fetch';
+// import fetch from 'isomorphic-fetch';
```

Comment:
Remove commented-out code here and elsewhere? Unless needed for reference while it's WIP